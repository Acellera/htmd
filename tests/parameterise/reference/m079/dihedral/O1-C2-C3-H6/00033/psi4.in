set {
	reference rhf
	basis 6-31G*
}


set_num_threads( 1 )
memory 2 gb

molecule MOL {
0 1

H	 -2.421521	 1.263602	 2.183376
C	 -2.069970	 0.240925	 2.062203
O	 -0.798221	 0.203235	 1.499556
H	 -2.009924	 -0.226204	 3.035510
H	 -2.792893	 -0.302773	 1.455406
C	 -0.719715	 0.768384	 0.229244
C	 0.719720	 0.768376	 -0.229267
H	 -1.321027	 0.195073	 -0.477462
H	 -1.093752	 1.790172	 0.236370
O	 1.392611	 -0.295398	 0.366430
H	 0.750534	 0.687200	 -1.313918
H	 1.185043	 1.714359	 0.050526
C	 2.706965	 -0.449871	 -0.062409
H	 2.756559	 -0.644779	 -1.131878
H	 3.305899	 0.433456	 0.155466
H	 3.125548	 -1.294566	 0.467107

	symmetry c1
}
set optking {
	frozen_dihedral = ("
		7 10 13 14,
		2 3 6 7
	")
}

ee = energy('scf')
oeprop('DIPOLE', 'QUADRUPOLE', 'MULLIKEN_CHARGES')
f=open( 'psi4out.xyz', 'w' )
f.write( "16  " )
f.write( str(ee) + "\n" )
f.write( MOL.save_string_xyz() )
f.close()
