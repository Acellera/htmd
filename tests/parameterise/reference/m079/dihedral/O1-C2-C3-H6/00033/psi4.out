    -----------------------------------------------------------------------
          Psi4: An Open-Source Ab Initio Electronic Structure Package
                              Psi4 (no tag) Driver

                          Git: Rev {detached?}  

    J. M. Turney, A. C. Simmonett, R. M. Parrish, E. G. Hohenstein,
    F. A. Evangelista, J. T. Fermann, B. J. Mintz, L. A. Burns, J. J. Wilke,
    M. L. Abrams, N. J. Russ, M. L. Leininger, C. L. Janssen, E. T. Seidl,
    W. D. Allen, H. F. Schaefer, R. A. King, E. F. Valeev, C. D. Sherrill,
    and T. D. Crawford, WIREs Comput. Mol. Sci. 2, 556-565 (2012)
    (doi: 10.1002/wcms.93)

                         Additional Contributions by
    A. E. DePrince, M. Saitow, U. Bozkaya, A. Yu. Sokolov
    -----------------------------------------------------------------------


    Psi4 started on: Tue Jul  5 16:36:21 2016

    Process ID:  28447
    PSI4DATADIR: /export131/home/mjharvey/miniconda3/share
    Memory level set to 256.000 MB

  ==> Input File <==

--------------------------------------------------------------------------
set {
	reference rhf
	basis 6-31G*
}


set_num_threads( 1 )
memory 2 gb

molecule MOL {
0 1

H	 -2.421521	 1.263602	 2.183376
C	 -2.069970	 0.240925	 2.062203
O	 -0.798221	 0.203235	 1.499556
H	 -2.009924	 -0.226204	 3.035510
H	 -2.792893	 -0.302773	 1.455406
C	 -0.719715	 0.768384	 0.229244
C	 0.719720	 0.768376	 -0.229267
H	 -1.321027	 0.195073	 -0.477462
H	 -1.093752	 1.790172	 0.236370
O	 1.392611	 -0.295398	 0.366430
H	 0.750534	 0.687200	 -1.313918
H	 1.185043	 1.714359	 0.050526
C	 2.706965	 -0.449871	 -0.062409
H	 2.756559	 -0.644779	 -1.131878
H	 3.305899	 0.433456	 0.155466
H	 3.125548	 -1.294566	 0.467107

	symmetry c1
}
set optking {
	frozen_dihedral = ("
		7 10 13 14,
		2 3 6 7
	")
}

ee = energy('scf')
oeprop('DIPOLE', 'QUADRUPOLE', 'MULLIKEN_CHARGES')
f=open( 'psi4out.xyz', 'w' )
f.write( "16  " )
f.write( str(ee) + "\n" )
f.write( MOL.save_string_xyz() )
f.close()
--------------------------------------------------------------------------

  Memory set to   2.000 GiB by Python script.

*** tstart() called on cx1-3-2-2.cx1.hpc.ic.ac.uk
*** at Tue Jul  5 16:36:22 2016


         ---------------------------------------------------------
                                   SCF
            by Justin Turney, Rob Parrish, and Andy Simmonett
                              RHF Reference
                        1 Threads,   2000 MiB Core
         ---------------------------------------------------------

  ==> Geometry <==

    Molecular point group: c1
    Full point group: C1

    Geometry (in Angstrom), charge = 0, multiplicity = 1:

       Center              X                  Y                   Z               Mass       
    ------------   -----------------  -----------------  -----------------  -----------------
           H         -2.628556604466     1.062604482468     1.533417539435     1.007825032070
           C         -2.277005604466     0.039927482468     1.412244539435    12.000000000000
           O         -1.005256604466     0.002237482468     0.849597539435    15.994914619560
           H         -2.216959604466    -0.427201517532     2.385551539435     1.007825032070
           H         -2.999928604466    -0.503770517532     0.805447539435     1.007825032070
           C         -0.926750604466     0.567386482468    -0.420714460565    12.000000000000
           C          0.512684395534     0.567378482468    -0.879225460565    12.000000000000
           H         -1.528062604466    -0.005924517532    -1.127420460565     1.007825032070
           H         -1.300787604466     1.589174482468    -0.413588460565     1.007825032070
           O          1.185575395534    -0.496395517532    -0.283528460565    15.994914619560
           H          0.543498395534     0.486202482468    -1.963876460565     1.007825032070
           H          0.978007395534     1.513361482468    -0.599432460565     1.007825032070
           C          2.499929395534    -0.650868517532    -0.712367460565    12.000000000000
           H          2.549523395534    -0.845776517532    -1.781836460565     1.007825032070
           H          3.098863395534     0.232458482468    -0.494492460565     1.007825032070
           H          2.918512395534    -1.495563517532    -0.182851460565     1.007825032070

  Running in c1 symmetry.

  Rotational constants: A =      0.34292  B =      0.05673  C =      0.05079 [cm^-1]
  Rotational constants: A =  10280.47171  B =   1700.77739  C =   1522.66548 [MHz]
  Nuclear repulsion =  261.204968331463760

  Charge       = 0
  Multiplicity = 1
  Electrons    = 50
  Nalpha       = 25
  Nbeta        = 25

  ==> Algorithm <==

  SCF Algorithm Type is DF.
  DIIS enabled.
  MOM disabled.
  Fractional occupation disabled.
  Guess Type is CORE.
  Energy threshold   = 1.00e-06
  Density threshold  = 1.00e-06
  Integral threshold = 0.00e+00

  ==> Primary Basis <==

  Basis Set: 6-31G*
    Number of shells: 56
    Number of basis function: 110
    Number of Cartesian functions: 110
    Spherical Harmonics?: false
    Max angular momentum: 2

  ==> Pre-Iterations <==

   -------------------------------------------------------
    Irrep   Nso     Nmo     Nalpha   Nbeta   Ndocc  Nsocc
   -------------------------------------------------------
     A        110     110       0       0       0       0
   -------------------------------------------------------
    Total     110     110      25      25      25       0
   -------------------------------------------------------

 OEINTS: Overlap, kinetic, potential, dipole, and quadrupole integrals
         stored in file 35.

  ==> Integral Setup <==

  ==> DFJK: Density-Fitted J/K Matrices <==

    J tasked:                  Yes
    K tasked:                  Yes
    wK tasked:                  No
    OpenMP threads:              1
    Integrals threads:           1
    Memory (MB):              1430
    Algorithm:                Core
    Integral Cache:           NONE
    Schwarz Cutoff:          1E-12
    Fitting Condition:       1E-12

   => Auxiliary Basis Set <=

  Basis Set: 
    Number of shells: 234
    Number of basis function: 736
    Number of Cartesian functions: 736
    Spherical Harmonics?: false
    Max angular momentum: 3

  Minimum eigenvalue in the overlap matrix is 3.6219608988E-03.
  Using Symmetric Orthogonalization.
  SCF Guess: Core (One-Electron) Hamiltonian.

  ==> Iterations <==

                           Total Energy        Delta E     RMS |[F,P]|

   @DF-RHF iter   1:  -240.55691127047930   -2.40557e+02   7.78125e-02 
   @DF-RHF iter   2:  -213.09487840841336    2.74620e+01   6.20564e-02 DIIS
   @DF-RHF iter   3:  -270.08971057960207   -5.69948e+01   5.14552e-02 DIIS
   @DF-RHF iter   4:  -305.24412325194965   -3.51544e+01   1.78429e-02 DIIS
   @DF-RHF iter   5:  -306.65908075577840   -1.41496e+00   7.56476e-03 DIIS
   @DF-RHF iter   6:  -306.95434927220856   -2.95269e-01   1.29441e-03 DIIS
   @DF-RHF iter   7:  -306.96712869204816   -1.27794e-02   3.02633e-04 DIIS
   @DF-RHF iter   8:  -306.96794945402621   -8.20762e-04   1.18003e-04 DIIS
   @DF-RHF iter   9:  -306.96805048840929   -1.01034e-04   5.07427e-05 DIIS
   @DF-RHF iter  10:  -306.96806996682437   -1.94784e-05   1.82977e-05 DIIS
   @DF-RHF iter  11:  -306.96807253039697   -2.56357e-06   1.95815e-06 DIIS
   @DF-RHF iter  12:  -306.96807257445687   -4.40599e-08   5.30694e-07 DIIS

  ==> Post-Iterations <==

    Orbital Energies (a.u.)
    -----------------------

    Doubly Occupied:                                                      

       1A    -20.550227     2A    -20.550192     3A    -11.269230  
       4A    -11.268582     5A    -11.261430     6A    -11.261427  
       7A     -1.388308     8A     -1.364179     9A     -1.030323  
      10A     -0.931476    11A     -0.897121    12A     -0.816882  
      13A     -0.688183    14A     -0.676057    15A     -0.656504  
      16A     -0.640649    17A     -0.616600    18A     -0.588259  
      19A     -0.572621    20A     -0.519380    21A     -0.506975  
      22A     -0.470963    23A     -0.465217    24A     -0.420014  
      25A     -0.401143  

    Virtual:                                                              

      26A      0.227705    27A      0.251539    28A      0.283747  
      29A      0.290486    30A      0.302179    31A      0.320104  
      32A      0.330160    33A      0.334000    34A      0.340666  
      35A      0.346490    36A      0.371602    37A      0.383469  
      38A      0.429097    39A      0.455799    40A      0.504237  
      41A      0.711307    42A      0.761194    43A      0.775287  
      44A      0.779974    45A      0.783610    46A      0.803248  
      47A      0.813661    48A      0.830643    49A      0.910865  
      50A      0.933933    51A      1.010104    52A      1.022693  
      53A      1.060332    54A      1.098120    55A      1.120400  
      56A      1.144651    57A      1.155234    58A      1.158646  
      59A      1.161480    60A      1.171803    61A      1.204355  
      62A      1.227172    63A      1.246167    64A      1.266216  
      65A      1.275730    66A      1.324328    67A      1.372859  
      68A      1.440972    69A      1.463425    70A      1.630184  
      71A      1.630289    72A      1.676497    73A      1.682096  
      74A      1.746090    75A      1.793690    76A      1.859001  
      77A      1.902890    78A      1.919379    79A      2.068103  
      80A      2.116363    81A      2.214906    82A      2.225826  
      83A      2.329012    84A      2.382041    85A      2.382588  
      86A      2.444953    87A      2.451135    88A      2.472035  
      89A      2.482335    90A      2.517163    91A      2.542852  
      92A      2.603382    93A      2.615505    94A      2.679924  
      95A      2.710109    96A      2.801518    97A      2.855082  
      98A      2.860867    99A      2.943845   100A      3.039546  
     101A      3.118822   102A      3.272071   103A      3.319030  
     104A      3.381083   105A      4.312962   106A      4.496135  
     107A      4.628609   108A      4.688864   109A      4.689228  
     110A      4.909631  

    Final Occupation by Irrep:
              A 
    DOCC [    25 ]

  Energy converged.

  @DF-RHF Final Energy:  -306.96807257445687

   => Energetics <=

    Nuclear Repulsion Energy =            261.2049683314637605
    One-Electron Energy =                -936.7221719341860080
    Two-Electron Energy =                 368.5491310282660038
    DFT Exchange-Correlation Energy =       0.0000000000000000
    Empirical Dispersion Energy =           0.0000000000000000
    PCM Polarization Energy =               0.0000000000000000
    EFP Energy =                            0.0000000000000000
    Total Energy =                       -306.9680725744562437

    Alert: EFP and PCM quantities not currently incorporated into SCF psivars.

Properties will be evaluated at   0.000000,   0.000000,   0.000000 Bohr
  ==> Properties <==


Properties computed using the SCF density matrix

  Nuclear Dipole Moment: (a.u.)
     X:    -0.5484      Y:     1.5028      Z:    -1.7214

  Electronic Dipole Moment: (a.u.)
     X:     0.3651      Y:    -1.0007      Z:     1.1462

  Dipole Moment: (a.u.)
     X:    -0.1832      Y:     0.5021      Z:    -0.5752     Total:     0.7852

  Dipole Moment: (Debye)
     X:    -0.4657      Y:     1.2762      Z:    -1.4620     Total:     1.9958


  Saving occupied orbitals to File 180.

*** tstop() called on cx1-3-2-2.cx1.hpc.ic.ac.uk at Tue Jul  5 16:36:29 2016
Module time:
	user time   =       6.43 seconds =       0.11 minutes
	system time =       0.05 seconds =       0.00 minutes
	total time  =          7 seconds =       0.12 minutes
Total time:
	user time   =       6.43 seconds =       0.11 minutes
	system time =       0.05 seconds =       0.00 minutes
	total time  =          7 seconds =       0.12 minutes


Properties will be evaluated at   0.000000,   0.000000,   0.000000 Bohr

Properties computed using the  density matrix

  Nuclear Dipole Moment: (a.u.)
     X:    -0.5484      Y:     1.5028      Z:    -1.7214

  Electronic Dipole Moment: (a.u.)
     X:     0.3651      Y:    -1.0007      Z:     1.1462

  Dipole Moment: (a.u.)
     X:    -0.1832      Y:     0.5021      Z:    -0.5752     Total:     0.7852

  Dipole Moment: (Debye)
     X:    -0.4657      Y:     1.2762      Z:    -1.4620     Total:     1.9958

  Quadrupole Moment: (Debye Ang)
    XX:   -31.2901     YY:   -37.9154     ZZ:   -36.6571
    XY:    -0.8800     XZ:    -2.6532     YZ:    -0.3374

  Traceless Quadrupole Moment: (Debye Ang)
    XX:     3.9975     YY:    -2.6278     ZZ:    -1.3696
    XY:    -0.8800     XZ:    -2.6532     YZ:    -0.3374

  Mulliken Charges: (a.u.)
   Center  Symbol    Alpha    Beta     Spin     Total
       1     H     0.43231  0.43231  0.00000  0.13537
       2     C     3.07890  3.07890  0.00000 -0.15780
       3     O     4.29276  4.29276  0.00000 -0.58551
       4     H     0.41076  0.41076  0.00000  0.17847
       5     H     0.43100  0.43100  0.00000  0.13801
       6     C     2.99400  2.99400  0.00000  0.01199
       7     C     2.99400  2.99400  0.00000  0.01200
       8     H     0.42944  0.42944  0.00000  0.14113
       9     H     0.43083  0.43083  0.00000  0.13834
      10     O     4.29276  4.29276  0.00000 -0.58552
      11     H     0.43083  0.43083  0.00000  0.13834
      12     H     0.42944  0.42944  0.00000  0.14113
      13     C     3.07890  3.07890  0.00000 -0.15779
      14     H     0.43232  0.43232  0.00000  0.13537
      15     H     0.43100  0.43100  0.00000  0.13800
      16     H     0.41076  0.41076  0.00000  0.17848

   Total alpha = 25.00000, Total beta = 25.00000, Total charge =  0.00000


*** PSI4 exiting successfully. Buy a developer a beer!
